freeipa/src/main/java/com/sequenceiq/freeipa/entity/UserSyncStatus.java

```diff
@@ -32,6 +32,12 @@
 
     private Long lastFullSyncEndTime;
 
+    @OneToOne
+    private Operation lastRequestedFullSync;
+
+    @OneToOne
+    private Operation lastSuccessfulFullSync;
```

Comment:
So the idea is that this holds the last successful operation and the lastRequestedFullSync holds the last full sync? Maybe rename it to reflect that (drop the requested?)